### ISSUE TITLE
About Homebrew Formula Ownership

### ISSUE THREAD

**Issue by weyderfs** [tgenv/tgenv#15]
Guys,
I've stared this [discussion](https://github.com/orgs/Homebrew/discussions/4125) at the Homebrew project about possibility transfer ownership of the TGEnv for us.

**Comment by weyderfs** [comment]
I will keep this _issue_ open for us and planning to following like the @iokiwi wrote in the [discussion](https://github.com/orgs/Homebrew/discussions/4125#discussioncomment-4748681) there.

**Comment by gtmtech** [comment]
I realise I'm not adding anything but even the tgenv shipped with homebrew is out of date in both this repo and the original repo at https://github.com/cunymatthieu/tgenv/

Would be super helpful to have this transferred and published at homebrew

**Comment by gtmtech** [comment]
Reading the comment on the discussion - did you ever publish to `tgenv/tgenv` - seems not?

```
brew install tgenv/tgenv
Warning: No available formula with the name "tgenv/tgenv".
```

**Comment by weyderfs** [comment]
> Reading the comment on the discussion - did you ever publish to `tgenv/tgenv` - seems not?
> 
> ```
> brew install tgenv/tgenv
> Warning: No available formula with the name "tgenv/tgenv".
> ```

Exactly @gtmtech, the available homebrew formula is not the one of our project. Recommend using the installation methods we have in the [README.md](https://github.com/tgenv/tgenv#installation-wrench).

**Comment by iokiwi** [comment]
Hey weyderfs. I'm going to have another go at moving this along. I'm not sure what homebrews policy is but enough time has passed and we have made reasonable attempts so I think we should be able to get support from the homebrew team

**Comment by iokiwi** [comment]
Yo, it looks like the home brew formula now points at our repo
https://formulae.brew.sh/formula/tgenv#default

**Comment by iokiwi** [comment]
Yep homebrew officially recognises our fork as the canonical version of tgenv now. 🥳 
https://github.com/Homebrew/homebrew-core/commit/6f73bf13a397e6f2e89a371a0cb835aa95a50362

I will close this discussion as it think it can be considered resolved. Note we have an open MR about adding the Formula to our repo that needs to be addressed.

https://github.com/tgenv/tgenv/pull/43

Personally I need to get my head around how one goes about maintaining a home brew formula

**Comment by weyderfs** [comment]
> Yep homebrew officially recognises our fork as the canonical version of tgenv now. 🥳 [Homebrew/homebrew-core@6f73bf1](https://github.com/Homebrew/homebrew-core/commit/6f73bf13a397e6f2e89a371a0cb835aa95a50362)
> 
> I will close this discussion as it think it can be considered resolved. Note we have an open MR about adding the Formula to our repo that needs to be addressed.
> 
> [#43](https://github.com/tgenv/tgenv/pull/43)
> 
> Personally I need to get my head around how one goes about maintaining a home brew formula

Great!!!
Thanks for closing @iokiwi the issue.
